pyproject.toml

```diff
@@ -108,6 +113,9 @@ strict = [
     "sevenn==0.9.3",
     "torch==2.2.1",
     "typing-extensions==4.12.2",
+    "mdanalysis==2.7.0",
+    "openmm>=8.1.0",
```

Comment:
This should be pinned with `==`. Also there is a new version available (`8.1.1`)

Reply:
What in the world..

Reply:
@peastman very curious what was the motivation behind removing old versions from pypyi?

Reply:
Are we sure there were prior PyPI versions to begin with? :)

Reply:
There weren't any.  The old package was an ancient front end for running simulations.  It hadn't been supported in many years.

The new packages are still experimental and shouldn't be relied on.  We're still exploring ways of creating PyPI packages.